### DISCUSSION TITLE
Deploy request stuck in queue

### DISCUSSION THREAD

**Discussion by jnewmano** [planetscale/discussion#165] [Bug Reports] [answered]
I have a deploy request that is stuck in the deploy queue, its current state is `closed` with a deploy state of `complete_error`. It is blocking all other deploy requests. Is there a way to remove it from the deploy queue?

**Comment by jnewmano** [comment, marked as answer]
Screenshot of UI showing current state of the deploy queue. Deploy Request #43 is closed and exited with an error.

![Deploy Queue Stuck](https://user-images.githubusercontent.com/6687368/154318403-4fd3d7ae-74c1-4640-a88d-de68ad9ed05c.png)

**Reply by orware** [reply]
Good morning Jason,

We wrapped up your related ticket over at PlanetScale Support yesterday so I hope that met your needs for this issue.

For a little extra background for those running into something similar, there is a related fix that will be coming soon (basically, for failed Deploy Requests there is a close button that wasn't initiating an extra action that's needed to coax things past the failed state, and this was preventing additional Deploy Requests from being processed correctly...for a few of these we've had to manually coax them past that state, but the fix will make that unnecessary). 

For anyone running into the issue between now and when the fix is rolled out, please reach out to us at PlanetScale Support (https://planetscale.com/support) and we'll be able to get things sorted out for you.

Please don't hesitate to reach back out to us if needed!

**Comment by Tor1mo** [comment]
Similar thing happening here. Deploy has been stuck in the queue for an hour. Can't cancel, can't delete the dev branch, can't create new deploys. Support ticket number #2907.

<img width="798" alt="image" src="https://user-images.githubusercontent.com/13493544/209429599-98824180-e5f6-4fe4-a7f3-313544479f6d.png">

**Reply by fooforge** [reply]
I have just responded to that ticket. You should be good to go to deploy that deploy request now.

**Reply by Tor1mo** [reply]
Thank you, Mike! That solved it 👍

**Reply by RaphaPlato** [reply]
Same here. What was the resolution? Was it to reach out to support?

**Reply by jnewmano** [reply]
Yeah, Planetscale had to do something on their end.

**Reply by RaphaPlato** [reply]
Created ticket #4215. FWIW, I had started 2 migrations in parallel on 2 different databases from the UI. I generally do both sequentially and don't have any issues.
